app/components/search/SearchResults/SearchResults.jsx

```diff
@@ -24,23 +26,54 @@ const transformHits = hits => hits.map(hit => {
   return { ...hit, recurringSchedule };
 });
 
-const SearchResults = ({ searchResults }) => {
+// Todo: setExpandList will be used as part of next stage of multiple location work
+// eslint-disable-next-line no-unused-vars
+const SearchResults = ({ searchResults, expandList, setExpandList }) => {
   if (!searchResults) return null;
-
+  const [centerCoords, setCenterCoords] = useState(null);
+  const [googleMapObject, setMapObject] = useState(null);
   const hits = transformHits(searchResults.hits);
 
+  useEffect(() => {
+    if (centerCoords) {
+      googleMapObject.setCenter(centerCoords);
+    }
+  }, [centerCoords]);
+
   return (
-    <div className={styles.searchResultsContainer}>
-      <div>
-        { hits.map((hit, index) => <SearchResult hit={hit} index={index} key={hit.id} />) }
+    <div className={styles.searchMapContainer}>
```

Comment:
Maybe rename this to `searchResultsAndMapContainer`? When I was reading the CSS file, I thought this only contained the map, but it actually contains both the map and the text results.

Reply:
Thanks for the call out! Changed it.